component/custom/custom.go

```diff
@@ -0,0 +1,67 @@
+package custom
+
+import (
+	"os"
+	"path/filepath"
+	"plugin"
+	"strings"
+
+	"github.com/Azure/azure-storage-fuse/v2/common/log"
+	"github.com/Azure/azure-storage-fuse/v2/exported"
+	"github.com/Azure/azure-storage-fuse/v2/internal"
+)
+
+func init() {
+	// Environment variable for directory containing the plugin .so files.
+	pluginDirPath := os.Getenv("BLOBFUSE_PLUGIN_DIRECTORY")
+	if pluginDirPath == "" {
```

Comment:
I still feel instead of giving a directory path and a files list is bit complicated for the user.
If I just have the plugin list then I can set for e.g. "BLOBFUSE_PLUGIN_NAMES=/home/user1/testcomponent1.so;/home/user2/testcomponent2.so;/mnt/temp/testcomponent3.so". Directory to load all .so files from it is not safe and bug prone.

Reply:
updated.